internal/cmd/auth/activate-service-account/activate_service_account.go

```diff
@@ -100,17 +100,18 @@ func configureFlags(cmd *cobra.Command) {
 	cmd.Flags().String(serviceAccountTokenFlag, "", "Service account long-lived access token")
 	cmd.Flags().String(serviceAccountKeyPathFlag, "", "Service account key path")
 	cmd.Flags().String(privateKeyPathFlag, "", "RSA private key path. It takes precedence over the private key included in the service account key, if present")
-	cmd.Flags().String(tokenCustomEndpointFlag, "", "Custom endpoint for the token API, which is used to request access tokens when the service-account authentication is activated")
-	cmd.Flags().String(jwksCustomEndpointFlag, "", "Custom endpoint for the jwks API, which is used to get the json web key sets (jwks) to validate tokens when the service-account authentication is activated")
 }
 
 func parseInput(p *print.Printer, cmd *cobra.Command) *inputModel {
+	tokenCustomEndpoint := viper.GetString(config.TokenCustomEndpointKey)
+	jwksCustomEndpoint := viper.GetString(config.JwksCustomEndpointKey)
+
 	model := inputModel{
 		ServiceAccountToken:   flags.FlagToStringValue(p, cmd, serviceAccountTokenFlag),
 		ServiceAccountKeyPath: flags.FlagToStringValue(p, cmd, serviceAccountKeyPathFlag),
 		PrivateKeyPath:        flags.FlagToStringValue(p, cmd, privateKeyPathFlag),
-		TokenCustomEndpoint:   flags.FlagToStringValue(p, cmd, tokenCustomEndpointFlag),
-		JwksCustomEndpoint:    flags.FlagToStringValue(p, cmd, jwksCustomEndpointFlag),
+		TokenCustomEndpoint:   tokenCustomEndpoint,
+		JwksCustomEndpoint:    jwksCustomEndpoint,
```

Comment:
For the other custom endpoints (e.g. the services) we don't add them to the input model of the command, so for uniformity I would follow the same approach here (and just use the values directly when needed)

Reply:
Alternatively we could consider adding the relevant custom endpoint (if any) to each command's input model

Reply:
I have removed the endpoints from the input model and carried them to storeFlags function